repl/src/test/resources/scriptLevelCaching/runTimeExceptions.scala

```diff
@@ -0,0 +1,4 @@
+import ammonite.ops._
+println(5/read(cwd/'repl/'src/'test/'resources/'scriptLevelCaching/"num.value").trim.toInt)
```

Comment:
You should make this read and write from `repl/target/scriptCachingTests/`, not the `src/` folder which is for source code